Modules/_threadmodule.c

```diff
@@ -1145,6 +1145,11 @@ thread_PyThread_start_new_thread(PyObject *self, PyObject *fargs)
         return NULL;
     }
 
+    if (PySys_Audit("_thread.start_new_thread", "OOO",
```

Comment:
this one doesn't appear to be mentioned in the docs.

Reply:
It's in the `_thread` docs, which is the most accurate but possibly the least helpful place to put it. But the main value of the docs is the table, so it'll appear there as the internal event that it is.